bandit/bandit.js

```diff
@@ -25,6 +26,11 @@ module.exports = (directory, inputFiles, params) => {
   const reportPath = path.join(directory, params.reportFile)
 
   let banditArgs = [...pyFiles, '--format', 'json', '-o', params.reportFile]
+
+  if (fs.existsSync(path.resolve('.bandit'))) {
+    banditArgs.push('-c', path.resolve('.bandit'))
```

Comment:
This can cause problems.  A .bandit file permits setting the targets (files), which would conflict with what we're trying to scan in the PR.

```
    ini_options = _get_options_from_ini(args.ini_path, args.targets)
    if ini_options:
        # prefer command line, then ini file
        args.excluded_paths = _log_option_source(args.excluded_paths,
                                                 ini_options.get('exclude'),
                                                 'excluded paths')

        args.skips = _log_option_source(args.skips, ini_options.get('skips'),
                                        'skipped tests')

        args.tests = _log_option_source(args.tests, ini_options.get('tests'),
                                        'selected tests')
        ini_targets = ini_options.get('targets')
        if ini_targets:
            ini_targets = ini_targets.split(',')
        args.targets = _log_option_source(args.targets, ini_targets,
                                          'selected targets')
```